### ISSUE TITLE
Re-add information about commit in docker build

### ISSUE THREAD

**Issue by andreeaflorescu** [rust-vmm/rust-vmm-container#74]
As part of #73 we are removing the information about the commit on which we build the crate. We should find a way to read this from the github workflow and re-add that information to the container for reproducibility purposes.

**Comment by stefano-garzarella** [comment]
@andreeaflorescu with https://github.com/rust-vmm/rust-vmm-container/pull/121 we now use the git sha1 as tag, so should be really straightforward to figure out how to reproduce an image of the container. Was that what you meant by this issue? Can we close it?

**Comment by andreeaflorescu** [comment]
Yes, what I wanted is to be able to reproduce a container build from a git commit. Closing this.
